### DISCUSSION TITLE
浅谈落实八小时工作制的益处

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#12] [Announcements]
# 浅谈落实八小时工作制的益处

本文是本人关于落实八小时工作制意义的一点浅薄理解。 先讲点题外话。 资本主义语境下对劳动的异化 劳动塑造人类，人类文明因劳动而璀璨。在原始劳动中，

https://zexwoo.blog/posts/editorial/worker-lives-matter/

**Comment by qxsny** [comment]
躺平是少数人，人多了就是罢工了。
